src/Model/Enums_and_constants/AppConstants.java

```diff
@@ -0,0 +1,15 @@
+package Model.Enums_and_constants;
```

Comment:
name all packages into lower case: model, control etc.
do not use in packages names "_"
store enums into "enums" package, constants - into "constant" package or into the package, where these constants are used (for example directly into model package)